client/src/components/user_reviews/user_reviews.jsx

```diff
@@ -3,14 +3,66 @@ import UserReviewsEntry from '../user_reviews_entry/user_reviews_entry.jsx';
 const React = require('react');
 
 
-const UserReviews = (props) => {
-  return (
-    <ul>
-      {props.data.map((data) => {
-        return <UserReviewsEntry key={props.data.indexOf(data)} review={data} keywords={props.keywords} />;
-      })}
-    </ul>
-  );
-};
+class UserReviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      page: 1,
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.pageButton = this.pageButton.bind(this);
+  }
+
+  handleClick(e) {
+    this.setState({ page: parseInt(e.target.innerText) });
+  }
+
+  pageButton(pageNumber, currentPage, numberOfPages, pageOne) {
+    if (pageNumber === pageOne || pageNumber === numberOfPages) {
+      return true;
+    }
+    if (pageNumber < currentPage
+      && (currentPage - pageNumber) > 1
+      && (numberOfPages - pageNumber) !== 2) {
+      return false;
+    }
+    if (pageNumber > currentPage && (pageNumber - currentPage) > 1 && (pageNumber - 1) !== 2) {
```

Comment:
I don't understand the pageNumber < or > than currentPage conditional. If it's greater than or less than, couldn't it just be refactored once to !== ?